data/json/mutation_category.json

```diff
@@ -144,7 +144,7 @@
         "name" : "Medical","//" : "2-6 pain, same as Alpha--since specifically intended for medical applications.",
         "category" : "MEDICAL",
         "mutagen_message" : "You can feel the blood rushing through your veins and a strange, medicated feeling washes over your senses.",
-        "iv_message" : "You can feel the blood in your medication stream.  It's a strange feeling.",
+        "iv_message" : "You can feel the medication in your blood stream.  It's a strange feeling.",
```

Comment:
Nope, this was intentional.

By the time you cross the Medical threshold, you've got more chemicals in there than blood.